cmd/org/createorg.go

```diff
@@ -66,7 +66,7 @@ var (
 func init() {
 	CreateCmd.Flags().StringVarP(&description, "desc", "d",
 		"", "Apigee org description")
-	CreateCmd.Flags().StringVarP(&analyticsRegion, "reg", "r",
+	CreateCmd.Flags().StringVarP(&analyticsRegion, "reg", "",
```

Comment:
why was this removed? Is it because of the global `r` flag? If yes, then removing the existing `r` for `reg` is breaking change right?